wordle-web/src/components/GameVue.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <div class="about">
```

Comment:
1.) You don't need this div here
2.) This file should be under your views folder, since it's a page that you route to